vm/urls.py

```diff
@@ -8,4 +8,5 @@
     url(r'^success$', views.success, name='success'),
     url(r'^(?P<vm_id>[0-9]+)/delete$', views.delete_vm, name='delete'),
     url(r'^(?P<vm_id>[0-9]+)/edit$', views.edit, name='edit'),
+    url(r'^(?P<vm_id>[0-9]+)/info$', views.info, name='info'),
```

Comment:
I think that the trailing `/info` is not needed and this url should be fixed to `^(?P<vm_id>[0-9]+)`.